www/js/diary/infinite_scroll_list.js

```diff
@@ -266,21 +317,28 @@ angular.module('emission.main.diary.infscrolllist',['ui-leaflet',
             });
             console.log("After reading in the label controller, manualResultMap "+JSON.stringify($scope.manualResultMap), $scope.data.manualResultMap);
             $scope.infScrollControl.pipelineRange = pipelineRange;
-            $scope.infScrollControl.currentEnd = pipelineRange.end_ts;
+            $scope.infScrollControl.pipelineRange.start_date = moment(pipelineRange.start_ts*1000).format("YYYY-MM-DD");
+            $scope.infScrollControl.pipelineRange.end_date = moment(pipelineRange.end_ts*1000).format("YYYY-MM-DD");
+            $scope.infScrollControl.oldestLoadedTs = pipelineRange.end_ts;
+            $scope.infScrollControl.latestLoadedTs = pipelineRange.end_ts;
```

Comment:
are these supposed to be the same (`pipelineRange.end_ts`)?

Reply:
I think this is fine. These are the initial values set - but as soon as any trips are loaded they'll be adjusted.